pages/02-01-up-and-running.md

```diff
@@ -22,15 +22,93 @@ Along with installing LÖVE, you will need a text editor for creating Lua files
 
 When you launch LÖVE, (see instructions below on how to do that) you will be greeted with a friendly graphic and the text "NO GAME", meaning you are running the engine but didn't give it a game to load.
 
-### macOS and other posix systems
+## macOS
+
+Once you have downloaded the LÖVE binary for macOS (64-bit zipped), proceed to the "Downloads" folder and unzip the archive. You should now see an application called "love".
+
+![macOS-love](/images/02-01-macos-folder.png)
+
+macOS might show you a warning modal, because you are trying to open an application by an unverified developer. If so, right click on the application and choose "open" and "open" again in the following dialog. You should now be greeted by the no-game screen.
+
+### Starting your game
+
+The simplest way to start a LÖVE game is to drag the whole folder containing the game's source files (not just the main.lua file!) onto the application file.
+
+![macOS-demo](/images/02-01-macos-drop-folder.gif)
+
+This also works with .love files.
+
+### Using the terminal
+
+If you are familiar with the [Terminal](https://en.wikipedia.org/wiki/Terminal_(macOS)), you can use it as a more convenient method of starting games.
```

Comment:
You have an extra parenthesis/bracket at the end.

Reply:
That's weird. The brackets are part of the URL and it isn't an issue on github:

[Terminal](https://en.wikipedia.org/wiki/Terminal_(macOS))

Any idea how to escape the bracket for gitbook?

Reply:
Oh I see now. You have it written correctly, but the markdown parser is confused by the parentheses in the url. You can fix it by using url-safe HTML escape characters instead:

```
[Terminal](https://en.wikipedia.org/wiki/Terminal_&#40;macOS&#41;)
```